### ISSUE TITLE
can not select radio input in sweet-modal

### ISSUE THREAD

**Issue by kinghs** [adeptoas/sweet-modal-vue#6]
hi, is there a problem to use radio input in sweet-modal?

I write this:
```
<sweet-modal title="Test" ref="testRadio">
        <input type="radio" name="ccc" v-model="pick" value="a"> test
        <input type="radio" name="ccc" v-model="pick" value="b"> test2
</sweet-modal>
```

but I can not select any one, help! please!

**Comment by bluefirex** [comment]
This was fixed just yesterday in 960bef6b30fe10347b9ad77ce18d4f8abc50f4c2

**Comment by kinghs** [comment]
thank you!
